### ISSUE TITLE
Battery charging property is not being updated

### ISSUE THREAD

**Issue by TanR314** [linkfrg/ignis#49]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Related to dotfiles ? *

- [X] I'm sure the problem is related specifically to Ignis.

### Regression?

Yes

### Version

 ```
Ignis 0.3
Branch: 
Commit:  ()
```

### Distro

NixOS 25.05

### Description

```python
from ignis.services.upower import UPowerService

upower = UPowerService.get_default()

for i in upower.batteries:
    i.connect(
        "notify::charging", lambda x, y, i=i: print("Charging status: ", i.charging)
    )
```

This does notify, but for some reasons, i.charging always returns False...
However, I checked upower, it shows that my battery is charging...

I saw https://github.com/linkfrg/ignis/pull/10#issuecomment-2394963596 .
But I didn't see any fixes

**Comment by linkfrg** [comment]
Probably Nix package related issue (?)

Can you try installing archlinux and test UPowerService on it?

**Comment by TanR314** [comment]
I installed Arch and tested it, but it doesn't work on Arch either.

**Comment by linkfrg** [comment]
Hmm, I will try to check it on the laptop soon

**Comment by linkfrg** [comment]
should be fixed after #59
